in_toto/in_toto_run.py

```diff
@@ -27,6 +27,40 @@
       -- vi foo.py
   ```
 
+<Arguments>
+  step_name:
+          A unique name to relate link metadata with a step defined in the
+          layout.
+  material_list:
```

Comment:
Should `material_list` be `materials`?  Please verify the names of the other command-line options, like `products` instead of `product_list`.

Reply:
Yes, I can change this. We actually don't have a guideline for documenting arpgarse arguments. 

There are several ways:
 - an *abstract* name for the argument (that's what i did),
 - the argument name(s) the command line user has to use (e.g. `-n/--step-name`), which only works for positional arguments,
 - the variable where argparse stores argument (e.g. `step_name`),
 - a combination of above

What do you suggest?

Reply:
Another way would be to just copy-paste the output of the command's help message.
I think I'll do that, and I will probably extend the help messages a little bit.

Reply:
I suggest something like the following:

```
<Arguments>
    -f/--foo (required argument):
        blah blah.
    -b/--bar (optional argument):
        bam bam.
    step-name (positional argument):
        Not a nickname, but a stepname.
```

You can also copy-paste the output of the command's help message.